model_analyzer/analyzer.py

```diff
@@ -89,43 +89,10 @@ def profile(self, client, gpus):
                 f"Expected config of type {ConfigCommandProfile},"
                 " got {type(self._config)}.")
 
-        self._metrics_manager = MetricsManager(
-            config=self._config,
-            client=client,
-            server=self._server,
-            gpus=gpus,
-            result_manager=self._result_manager,
-            state_manager=self._state_manager)
+        self._create_profile_managers(client, gpus)
```

Comment:
Can we refactor this further into _create_metrics_manager & _create_model_manager?

Reply:
done